lib/pages/new_group/new_group_chat_info.dart

```diff
@@ -294,17 +294,8 @@ class NewGroupChatInfoController extends State<NewGroupChatInfo>
   }
 
   void _getDefaultGroupName(Set<PresentationContact> contactList) async {
```

Comment:
it makes more sense 

```suggestion
  void _initializeGroupName() {
```

Reply:
because i don't need it anymore, so removed it.